libs/template/file.go

```diff
@@ -53,6 +58,10 @@ type copyFile struct {
 	srcPath string
 }
 
+func (f *copyFile) isNotebook() bool {
+	return strings.HasSuffix(f.DstPath().relPath, ".ipynb")
+}
```

Comment:
If you're relying only on a path check, this can be done once as part of a `writeFile` routine.

Reply:
done